src/client/components/ConsentsHeader.tsx

```diff
@@ -14,7 +14,7 @@ type Props = {
 export const ConsentsHeader = ({ error, success, geolocation }: Props) => (
   <>
     <Header geolocation={geolocation} />
-    {error && <GlobalError error={error} link={getErrorLink(error)} left />}
+    {error && <GlobalError error={error} link={getErrorLink()} left />}
```

Comment:
Could `link` just be `DEFAULT_ERROR_LINK` and remove the `getErrorLink` function, as it only return a single value now ?